website/static/js/fangorn.js

```diff
@@ -88,6 +88,8 @@ var COMMAND_KEYS = [224, 17, 91, 93];
 var ESCAPE_KEY = 27;
 var ENTER_KEY = 13;
 
+var MILLISECONDS_PER_MOVE_REQUEST = 500;
```

Comment:
What is the standard for constants in js files?  Do we define at the top? Do we define in settings and import? I see some other variables defined here, just wondering what best practice.